uptime-kuma/egg-uptime-kuma.json

```diff
@@ -58,9 +62,21 @@
             "default_value": "1",
             "user_viewable": true,
             "user_editable": true,
-            "rules": "required|boolean",
-            "sort": null,
-            "field_type": "text"
+            "rules": [
+                "required",
+                "boolean"
+            ],
+            "sort": 3
+        },
+        {
+            "name": "pathspec",
```

Comment:
Not needed, becaue author only support lastest released version. No need to choose version